frontend/src/utils/config.js

```diff
@@ -3,6 +3,7 @@
 
 //TreeViewer.jsx config
 export const VConf = Object.freeze({
+  MAX_DEPTH: 2, //The maximum depth of nodes to be visible to the viewer in the tree
```

Comment:
Lol isn't this from my zoom MR? How come it is showing up as a change here ...

Reply:
I merged develop into mine and then resolved conflicts, and pushed. Might be why it has shown up. Does it have any purpose now, with the zooming?

Reply:
seems gone... :shrug: